docs/Materials.md.html

```diff
@@ -2131,21 +2131,15 @@
 
 ### Frame constants
 
-                 Name               |   Type   |            Description
-:-----------------------------------|:--------:|:------------------------------------
-**getResolution()**                 | float4   |  Resolution of the view in pixels: `width`, `height`, `1 / width`, `1 / height`
-**getWorldCameraPosition()**        | float3   |  Position of the camera/eye in world space
-**getWorldOffset()**                | float3   |  The shift required to obtain API-level world space
-**getTime()**                       | float    |  Current time as a remainder of 1 second. Yields a value between 0 and 1
-**getUserTime()**                   | float4   |  Current time in seconds: `time`, `(double)time - time`, `0`, `0`
-**getUserTimeMode(float m)**        | float    |  Current time modulo m in seconds
-**getExposure()**                   | float    |  Photometric exposure of the camera
-**getEV100()**                      | float    |  [Exposure value at ISO 100](https://en.wikipedia.org/wiki/Exposure_value) of the camera
-
-!!! TIP: world space
-    To achieve good precision, the "world space" in Filament's shading system does not necessarily
-    match the API-level world space. To obtain the position of the API-level camera, custom
-    materials can add `getWorldOffset()` to `getWorldCameraPosition()`.
+                 Name                   |   Type   |            Description
+:---------------------------------------|:--------:|:------------------------------------
```

Comment:
nit: why did you change column span?

Reply:
This is the result of multiple commits, will revert. Thanks for spotting!